static/app/views/performance/cache/tables/cacheHitMissCell.tsx

```diff
@@ -0,0 +1,15 @@
+import {t} from 'sentry/locale';
+import type {IndexedResponse} from 'sentry/views/starfish/types';
+
+function CacheHitMissCell(props: {hit: IndexedResponse['cache.hit']}) {
+  const {hit} = props;
+  if (hit === 'true') {
+    return <span>{t('HIT')}</span>;
+  }
+  if (hit === 'false') {
+    return <span>{t('MISS')}</span>;
+  }
+  return <span>--</span>;
+}
+
+export default CacheHitMissCell;
```

Comment:
```suggestion
export default CacheHitMissCell;
```

Better to stick to named exports, defaults are harder to track down, replace, etc.

Reply:
Fair! This sounds like a good contender for a lint warning